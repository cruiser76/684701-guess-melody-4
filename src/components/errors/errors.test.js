import React from 'react';
import renderer from 'react-test-renderer';

import Errors from './errors.jsx';

const userErrors = 3;

it(`Errors renders correctly`, () => {
  const tree = renderer
    .create(
        <Errors
          userErrors={userErrors}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
