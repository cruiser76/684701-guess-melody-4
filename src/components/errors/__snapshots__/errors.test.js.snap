// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors renders correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
